### PR TITLE
Add Reproducible Research

### PR DESCRIPTION
**Pull request by leipzig**
(and Information Science section)

**https://github.com/leipzig/awesome-reproducible-research**

**Reproducible research is a field of great interest to the physical and life sciences, statistics, data science, and computer science**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the project/theme of the list. **It should not describe the list itself.**<br>Example: `- [Fish](…) - User-friendly shell.`, not `- [Fish](…) - Resources for Fish.`.
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
	- Not a duplicate.
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Either fullwidth or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge).
		- Should be placed on the right side of the readme heading.
		- Should link back to this list.
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
		- Should only have one level of sub-lists, preferably none.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- [WTFPL](http://www.wtfpl.net) and [Unlicense](http://unlicense.org) are not acceptable licenses.
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
		- The description starts with an uppercase character and ends with a period.
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by legodude17** [comment]
> (all studies)

What does this mean?

**Comment by leipzig** [comment]
The Ioannidis paper "Why most published research findings are false" is a theoretical survey that applies to all scientific papers that produced a p-value.

**Comment by legodude17** [comment]
Ah thank you.

**Comment by sindresorhus** [comment]
Hey, sorry for the lack of response, please see #1363.

**Comment by sindresorhus** [comment]
> * The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.

File: readme.md [hunk 1]
```diff
@@ -284,7 +284,11 @@
 - [Open Source Society University](https://github.com/open-source-society/computer-science)
 - [Functional Programming](https://github.com/lucasviola/awesome-functional-programming)
 - [Static Analysis & Code Quality](https://github.com/mre/awesome-static-analysis)
+
+
+## Information Science
 - [Information Retrieval](https://github.com/harpribot/awesome-information-retrieval) - Learn to develop your own search engine.
+- [Reproducible Research](https://github.com/leipzig/awesome-reproducible-research) - Go from raw data to results in a repeatable and transparent manner
```
Comment (sindresorhus) [inline readme.md]: Description should end in a `.`

File: readme.md [hunk 2]
```diff
@@ -284,7 +284,11 @@
 - [Open Source Society University](https://github.com/open-source-society/computer-science)
 - [Functional Programming](https://github.com/lucasviola/awesome-functional-programming)
 - [Static Analysis & Code Quality](https://github.com/mre/awesome-static-analysis)
+
+
+## Information Science
```
Comment (sindresorhus) [inline readme.md]: Creating a category should be a separate PR after this one. You also need to add it to the `Contents` section.

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

**Comment by sindresorhus** [comment]
```
❯ awesome-lint https://github.com/leipzig/awesome-reproducible-research
✖ Linting

  readme.md:824:5
  ✖    1:34   Invalid badge source                                    remark-lint:awesome/badge
  ✖  824:5    Incorrect indentation before bullet: remove 2 spaces    remark-lint:list-item-bullet-indent
  ✖  825:5    Incorrect indentation before bullet: remove 2 spaces    remark-lint:list-item-bullet-indent
  ✖  827:5    Incorrect indentation before bullet: remove 2 spaces    remark-lint:list-item-bullet-indent
  ✖  828:5    Incorrect indentation before bullet: remove 2 spaces    remark-lint:list-item-bullet-indent
  ✖  829:5    Incorrect indentation before bullet: remove 2 spaces    remark-lint:list-item-bullet-indent
  ✖  833:5    Incorrect indentation before bullet: remove 2 spaces    remark-lint:list-item-bullet-indent
  ✖  834:5    Incorrect indentation before bullet: remove 2 spaces    remark-lint:list-item-bullet-indent
  ✖  835:5    Incorrect indentation before bullet: remove 2 spaces    remark-lint:list-item-bullet-indent
  ✖  843:72   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  849:38   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  850:35   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  854:104  List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  857:43   Text "Github" should be written as "GitHub"             remark-lint:awesome/spell-check
  ✖  860:41   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  861:50   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  864:55   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  865:35   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  866:43   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  870:69   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  871:65   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  872:72   List item description must end with proper punctuation  remark-lint:awesome/list-item
  ✖  873:63   List item description must end with proper punctuation  remark-lint:awesome/list-item

  23 errors
```

https://github.com/sindresorhus/awesome-lint

File: readme.md [hunk 3]
```diff
@@ -626,4 +626,4 @@
 
 [![CC0](http://mirrors.creativecommons.org/presskit/buttons/88x31/svg/cc-zero.svg)](https://creativecommons.org/publicdomain/zero/1.0/)
 
-To the extent possible under law, [Sindre Sorhus](https://sindresorhus.com) has waived all copyright and related or neighboring rights to this work.
+To the extent possible under law, [Sindre Sorhus](https://sindresorhus.com) has waived all copyright and related or neighboring rights to this work.
```
Comment (sindresorhus) [inline readme.md]: Don't do unrelated changes.

**Comment by sindresorhus** [comment]
https://github.com/sindresorhus/awesome/pull/1164/files#r260135208

**Comment by sindresorhus** [comment]
> The description starts with an uppercase character and ends with a period.

**Comment by sindresorhus** [comment]
> Includes a succinct description of the project/theme at the top of the readme. (Example)
